src/ToggleButton/Option.js

```diff
@@ -0,0 +1,253 @@
+// @flow
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ButtonBase from '../internal/ButtonBase';
+import Menu from '../Menu';
+import ArrowDropdown from '../svg-icons/arrow-drop-down';
+import withStyles from '../styles/withStyles';
+import createStyleSheet from '../styles/createStyleSheet';
+import { fade } from '../styles/colorManipulator';
+import common from '../colors/common';
+import grey from '../colors/grey';
+
+/**
+ * Represents an option within a Toggle Button
+ *
+ */
+
+export const styleSheet = createStyleSheet('MuiOption', {
+  root: {
+    backgroundColor: 'transparent',
+    borderRadius: 0,
+    fontWeight: 600,
+    fontSize: 14,
+    float: 'left',
+    padding: '0px 12px',
+    textTransform: 'uppercase',
+    // borderLeft: props.optionStyle.borderLeft? props.optionStyle.borderLeft : 'none',
+    // borderRight: props.optionStyle.borderRight? props.optionStyle.borderRight : 'none',
+    height: 36,
+  },
+  button: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    color: fade(common.black, 0.3),
+  },
+  dropDownButton: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    float: 'left', // TODO: Move this down to where the dropdown is being generated
+    color: fade(common.black, 0.3),
+  },
+  textSelected: {
+    color: fade(common.black, 0.54),
```

Comment:
When using _toggleIcon_ to present icons as toggle button options, the only indication that an icon has been selected is that the color changes slightly.  In the [specification](https://material.io/guidelines/components/buttons.html#buttons-toggle-buttons), the color of the icon is the same for normal, hover, focus, and pressed states.  The visual cue that an icon is in the pressed state is its radial background, which is a function of the icon's color.

>The icon toggle focus indicator color and opacity are related to the color of the icon.

For this to work for icons, there will have to be some work done to the background.  It would be nice to support a disabled icon as well.  Perhaps through a type property, similar to the one implemented for button (primary, accent, disabled, contrast, default).